ui/raidboss/data/02-arr/trial/titan-nm.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+[{
+  zoneId: ZoneId.TheNavel,
+  timelineFile: 'titan-nm.txt',
+  triggers: [
+    {
+      id: 'TitanNm Tumult',
+      netRegex: NetRegexes.ability({ id: '282', source: 'Titan', capture: false }),
+      condition: (data) => data.role === 'healer' || data.CanAddle(),

```

Comment:
As it happens, there's a built-in condition for this, [`caresAboutAOE()`](https://github.com/quisquous/cactbot/blob/a5d5de00566ce83529c52b172112fbfe45051b5b/resources/conditions.js#L13)!

Reply:
I didnt actually know about that, I will fix that. Thanks!